lib/index.js

```diff
@@ -170,7 +170,7 @@ module.exports = class ApiClient {
    * @return {ApiClient}
    */
   setVersion (version) {
-    if (!version) {
+    if (!version || version > 2 || isNaN(version)) {
```

Comment:
Since it's very specific to `1` or `2`, why not just do

```
if (version !== 1 && version !== 2) {
```
or
```
if (![1, 2].includes(version)) {
```